README.md

```diff
@@ -77,12 +100,64 @@ sqlite> SELECT path FROM cfgov_fts WHERE cfgov_fts MATCH 'diamonds' ORDER BY pat
 
 Note that this query uses a distinct `cfgov_fts` table that uses the SQLite [FTS5 extension](https://www.sqlite.org/fts5.html) for full-text search.
 
-## Development
+## Running the viewer application
+
+From the repo's root, compile front-end assets:
 
 ```
 yarn
-yarn start
+yarn build
+```
+
+Create a Python virtual environment and install requirements:
+
+```
+python3.8 -m venv venv
+source venv/bin/activate
+pip install -r viewer/requirements.txt
 ```
+Set required environment variable pointing to crawl database:
+
+```
+export CRAWL_DATABASE=cfgov.sqlite3
+```
+
+Finally, run the Django webserver:
+
+```
+viewer/manage.py runserver
+```
+
+The viewer application will be available locally at http://localhost:8000.
+
+## Development
+
+### Testing
+
+To run Python unit tests, use [`tox`](https://tox.wiki/en/latest/).
```

Comment:
Should `tox` be added to `requirements.txt`?

Reply:
I didn't add tox because it's only needed for testing and thus doesn't need to be installed e.g. on deploys. But for simplicity's sake we might as well add it. I'll do so, thanks!